dll/win32/shlwapi/ordinal.c

```diff
@@ -4121,6 +4121,32 @@ BOOL WINAPI IsOS(DWORD feature)
     return FALSE;
 }
 
+#ifdef __REACTOS__
+/*************************************************************************
+ * @  [SHLWAPI.438]
+ */
+HRESULT WINAPI SHLoadRegUIStringA(HKEY hkey, LPCSTR value, LPSTR buf, DWORD size)
```

Comment:
Usually the A version does do some conversion and calls the W version,
but this works fine for me as well.